docs/extensibility/README.md

```diff
@@ -21,9 +21,12 @@ To automatically add the UI page for your CRD, follow these steps:
 To create your CRD Config Map, follow these steps:
 
 1. Go to the `kube-public` Namespace and choose **Configuration > Config Maps**.
-2. Click **Create Config Map +** and enter the name of your Config Map.
-3. Under the **Data** form, you can add the required fields to define how to handle your CRD.
-4. Go to the **Advanced** tab and in the **Labels** form enter `busola.io/extension` as a key, and `resource` as a value.
+
+> **NOTE:** You can choose your Namespace, but the `kube-public` Namespace is preferred.
+
+1. Click **Create Config Map +** and enter the name of your Config Map.
+2. Under the **Data** form, you can add the required fields to define how to handle your CRD.
```

Comment:
If the information is required, I wouldn't use `can` (which implies "optional").
Then again, if the whole step is optional (but if done, the fields are mandatory), keep the `can` :)

```suggestion
2. Under **Data**, add the required fields to define how to handle your CRD.
```